### PR TITLE
Scrabble Score

### LINKED ISSUE #60: Scrabble Score

### PR DESCRIPTION
**Pull request by glennj**
Closes #60

### CODE REVIEW

File: exercises/practice/scrabble-score/.meta/example.awk [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    tile["J"]=8; tile["X"]=8
+    tile["Q"]=10; tile["Z"]=10
+
+    # FS cannot contain space
+    FS = OFS = ","
+}
+
+{
+    $1 = toupper($1)
+    split($1, letters, //)
```
Comment (IsaacG) [inline exercises/practice/scrabble-score/.meta/example.awk:19]: Can't you also just do `FS = ""` to get fixed point?

**Review by IsaacG — APPROVED** [review]

File: exercises/practice/scrabble-score/.meta/example.awk (hunk 1 above)
Comment (glennj) [reply exercises/practice/scrabble-score/.meta/example.awk:19]: Sure can.

Comment (glennj) [reply exercises/practice/scrabble-score/.meta/example.awk:19]: And from reading about `FPAT` for ocr-numbers, can also do `FPAT="[[:alpha:]]"` -- the numbered fields are ***only the letters in the string***.
```
$ echo 'abc123def' | gawk -v FPAT='[[:alpha:]]' '{for (i=1; i<=NF; i++) print i, $i}'
1 a
2 b
3 c
4 d
5 e
6 f
```
How cool is that?

Comment (IsaacG) [reply exercises/practice/scrabble-score/.meta/example.awk:19]: Wow! That's incredibly useful. Apparently there's also `FIELDWIDTHS` for reading in blocks of bytes.

**Pull request merged**